test_interpret.js fails on Windows

``` sh
./traceur test/unit/runtime/test_interpret.js
Failed to load 'd:../../../src/util/assert.js'

undefined:20182
        throw ex;
              ^
null
make: *** [test-interpret] Error 8
```

I haven't had time to look at this yet. How are we picking the fileLoader in this test?

--- a/src/node/interpreter.js
+++ b/src/node/interpreter.js
@@ -37,7 +37,7 @@ function interpret(filename, argv, flags) {
     return new traceur.modules.Loader(loaderHooks);
   }
   global.SystemLoader = getLoader();
-  global.SystemLoader.import(filename);
+  global.SystemLoader.import(filename.replace(/\\/g, '/'));
 }
 
 module.exports = interpret;

